footprints/main/views.py

```diff
@@ -317,8 +317,45 @@ def get_rows(self):
                    'Imprint Actor and Role', 'Imprint BHB Number',
                    'Imprint OCLC Number', 'Evidence Type', 'Evidence Location',
                    'Evidence Call Number', 'Evidence Details']
+        for r in Role.objects.for_footprint():
+            role = 'Footprint Role ' + unicode(r.name)\
```

Comment:
It looks like this will fail for strings containing unicode characters, for example:
```
>>> unicode('ábc')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)

>>> unicode('ábc').encode('utf-8')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)
```

I think what you're looking for might be the `smart_text` function that Django provides: https://docs.djangoproject.com/en/1.11/ref/utils/#module-django.utils.encoding  and it's compatible with python 2 and 3.